### ISSUE TITLE
NoFakeNews

### ISSUE THREAD

**Issue by kibatech** [MozillaFestival/open-leaders-7#95]
Project Lead:

Mentor:

Welcome to OL7, Cohort B! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Leadership Training :tada:.

***

Before Week 1 (Jan 29): Your first mentorship call
- [ ] Complete the OLF self-assessment ([online](https://mzl.la/olf-online-assessment), [printable](https://mzl.la/olf-self-assessment)). If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [ ] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (Feb 5): First Cohort Call (Open by Design)
- [ ] Complete Assignment ['Write Your Vision or Mission Statement'](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/). Leave a comment on this issue with your draft vision statement & be ready to share this on the call.
- [ ] Watch ['Mozilla & Working Open'](https://youtu.be/quKdaqlR_9w).
- [ ] Check the [Syllabus](https://mzl.la/ol7-syllabus) for notes and connection info for all the cohort calls.

Before Week 3 (Feb 12): Mentorship call
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement.
- [ ] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://goo.gl/to6PYn)). Comment on this issue with a link to your canvas.
- [ ] [Start your Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/). Comment on this issue with your draft Roadmap.

Before Week 4 (Feb 19): Cohort Call (Build for Understanding)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.
- [ ] [Pick an open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) for the work you're doing during the program.
- [ ] Use your canvas to [start writing a README](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project. Link to your README in a comment on this issue.

Week 5 and more
- [ ] Create a repository for your project. New to GitHub? You can [fork our templates](https://mozilla.github.io/global-sprint/project-lead-guide/templates/).
- [ ] Add [Contributing Guidlines](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/)
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/)
- [ ] [Submit your project to Mozilla Pulse](https://www.mozillapulse.org/add), add the tag `#mozsprint`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Leaders. Please refer to the [OL7 Syllabus](https://mzl.la/ol7-syllabus) for more detailed weekly notes and assignments past week 4.

**Comment by kibatech** [comment]
“Working on the internet to make it free of misinformation and fake news.”

**Comment by foretia** [comment]
> “Working on the internet to make it free of misinformation and fake news.”

Great vision, especially at an era where the spread of misinformation is very rampart

**Comment by veritoportugal** [comment]
Create visión.I would Like to see how to verify the news

**Comment by lpatmo** [comment]
> “Working on the internet to make it free of misinformation and fake news.”

Love this. Curious about the strategy!

**Comment by BhuvanaMeenakshiK** [comment]
> “Working on the internet to make it free of misinformation and fake news.”

This is cool!!

**Comment by BhuvanaMeenakshiK** [comment]
> > “Working on the internet to make it free of misinformation and fake news.”
> 
> This is cool!!

Would love to know on the accuracy on how well it will play a role like will it automated or will it be a manual check?

**Comment by kibatech** [comment]
Thank you for your interest in my project. We are looking at automating it.
It will take the form of a plugin that will validate news reports and check
for the authenticity of the reports.

**Comment by abhayrjoshi** [comment]
Hello Kibatech, Can you please elaborate on how you are going to automate it? How would the authenticity of the reports be determined?

**Comment by kibatech** [comment]
By fact checking the news report on reputable platforms and reliable
resources. We can grade the reports as

Reliable, Not too reliable, Yet to be verified, Unreliable.

We are still working on the schematics. We have not concluded yet.

**Comment by abhayrjoshi** [comment]
So, at times, the "reliable" and "reputable" sources also pick up the viral content that is moving around, with less focus towards authenticity. How do you tend to classify that?

**Comment by kibatech** [comment]
You are right but we will be very stringent in the choice of our platforms.
Also, when we will continuously review and improve our algorithms in a bid
to make sure they are addressing the focus challenge of Fake news.

**Comment by samcillabaakojr** [comment]
The spread of misinformation can cause several harms online, it's a great thought and start but how will your automation work in verifying fake news? Awesome vision by the way

**Comment by anacarolinards** [comment]
@abhayrjoshi did good questions. If the tool could bring tips and information educating the user on how to do their own checking, it would be great!

**Comment by kibatech** [comment]
Thanks for the insight and feedback. It means a lot to this project. I'm
looking for collaborators and resource persons that can support this
project.

I'm always open to new ideas.

Best regards.
Kayode.
